DEVELOPERS_GUIDE.md

```diff
@@ -0,0 +1,28 @@
+# Building for testing
+
+To test a branch in GitHub Actions, an updated `dist/index.js` file is required.
+
+```
+npm run build
+git add dist/.
+git commit -m "updating index.js"
+git log -q -n 1 dist/index.js | less -F
+```
+
+From the log output take note of the commit hash and push to GitHub
+
+In a test GitHub action you can use the branched build of the action by referencing the commit hash
+
+```
+- name: Create Zip Package for Octopus Deploy
+      # You may pin to the exact commit or the version.
+        uses: OctopusDeploy/create-zip-package-action@6d73e1f62030dfa23534ff0d279774177d5ea1ea
```

Comment:
```suggestion
        uses: OctopusDeploy/create-zip-package-action@my-branch
```
I've found it's much easier to work with just the branch name, not specific commits. You can run the workflow in the consuming repo then without requiring an update to the commit hash as you make changes.

Reply:
good point